Packs/Code42/Integrations/Code42/Code42.yml

```diff
@@ -805,7 +805,7 @@ script:
     - contextPath: Code42.DepartingEmployee.DepartureDate
       description: The departure date for the Departing Employee.
       type: Unknown
-  dockerimage: demisto/py42:1.0.0.9806
+  dockerimage: demisto/py42:1.0.0.10082
```

Comment:
I am not sure why some of these changes are duplicated

Reply:
What do you mean?

Reply:
This change is already present in the base branch... https://github.com/demisto/content/pull/8282